demos/dev_mode_key_provisioning/src/aws_dev_mode_key_provisioning.c

```diff
@@ -182,6 +182,7 @@ static CK_RV prvProvisionPrivateECKey( CK_SESSION_HANDLE xSession,
             { CKA_VALUE,     pxD,               EC_D_LENGTH                                      }
         };
 
+        configPRINTF( ( "Write Private EC Key...\r\n" ) );
```

Comment:
I'm finding myself distracted by the capitalization in these new comments. It looks like most of the existing comments use regular sentence casing.

Reply:
Noted, I will update all the statements to regular sentences